docs/cloud-providers/aws/aws-provider.md

```diff
@@ -41,7 +41,7 @@ resources.
 You can confirm the existense of the  AWS `ProviderConfig` by running:
 
 ```bash
-kubectl -n crossplane-system get provider/aws-provider
+kubectl -n crossplane-system get provider aws-provider
```

Comment:
Thanks for looking into this! I _think_ this might should actually look like the following with recent updates:
```suggestion
kubectl get providerconfig aws-provider
```

Reply:
@hasheddan Seems it's still not in the alpha release. I'm on `0.12.0`

```bash
❯ kubectl get providerconfig aws-provider
error: the server doesn't have a resource type "providerconfig"
```

Reply:
Yep, this commit is to master though, meaning it will be in the next release. Also, you’ll have to manually add DCO to that commit because the Github UI doesn’t do it for you :/

Reply:
yup, fixed 👍